### PR TITLE
228 refactor data management

### PR DESCRIPTION
**Pull request by daveatweaviate**
Gathering spot for refactor changes

### CODE REVIEW

File: developers/weaviate/manage-data/collections.mdx [hunk 1]
```diff
@@ -0,0 +1,469 @@
...
+- The vectorizer module (`img2vec-neural`)
+
+  - The `image` property configures collection to store image data.
+  
+- The vector index distance metric (`cosine`)
+- A set of properties (`image`), with the `image` property set as `blob`.
+
+For image searches, see [Image search](../search/image.md).
+
+</details>
```
Comment (sebawita) [inline developers/weaviate/manage-data/collections.mdx:459]: The sample configuration only provides Python v3 examples.

Can we add examples for Python v4 and TypeScript?

Comment (sebawita) [reply developers/weaviate/manage-data/collections.mdx:459]: Also, can we use code highlighting for the essential parts of the code.
For example, in the example showcasing generative configuration, we should highlight the 3 lines that show that configuration:
```
    "moduleConfig": {
        "generative-openai": {}
    },
```

Comment (daveatweaviate) [reply developers/weaviate/manage-data/collections.mdx:459]: @sebawita I thought json meant those were generic configs, that's why I moved them to a sample section.  I can open another ticket to update them

Comment (sebawita) [reply developers/weaviate/manage-data/collections.mdx]: sounds good 👍 
It looks like this got addressed in #1435 😄

**Pull request merged**